### PR TITLE
FS 1409 update application question forms

### PR DESCRIPTION
**Pull request by ramsharma-prog**
Application questions update as per the new prototype

### CODE REVIEW

**Review by harryyo — APPROVED** [review]
👍

**Review by adamdavies1 — CHANGES_REQUESTED** [review]

File: form_jsons/public/community-engagement.json [hunk 1]
```diff
@@ -50,22 +42,16 @@
...
           "type": "MultilineTextField",
           "title": "Tell us how your project supports any wider local plans"
         }
       ],
-      "next": [
-        {
-          "path": "/summary"
-        }
-      ]
+      "next": [{ "path": "/check-your-answers" }]
```
[the remark this answers is not in the document]
Comment (ramsharma-prog) [reply form_jsons/public/community-engagement.json:50]: Okay will amend it manually & add back link as well when finished updating other contents

[the remark this answers is not in the document]
Comment (ramsharma-prog) [reply form_jsons/public/community-engagement.json]: Done!

**Review by adamdavies1 — APPROVED** [review]
LGTM

**Pull request merged**